MOVR style updates

elkmaster opened the pull request:
![Screenshot 2020-07-21 at 19 22 37](https://user-images.githubusercontent.com/12850886/88080422-b8d1c480-cb87-11ea-959c-06c3d3752ab8.png)
![Screenshot 2020-07-21 at 19 22 44](https://user-images.githubusercontent.com/12850886/88080429-ba02f180-cb87-11ea-9949-0a39a00ba00d.png)
![Screenshot 2020-07-21 at 19 21 37](https://user-images.githubusercontent.com/12850886/88080457-c4bd8680-cb87-11ea-9212-b8d80fbd23f2.png)
![Screenshot 2020-07-21 at 19 22 06](https://user-images.githubusercontent.com/12850886/88080443-bf603c00-cb87-11ea-9f78-96bbeaea4f3a.png)
![Screenshot 2020-07-21 at 19 21 50](https://user-images.githubusercontent.com/12850886/88080464-c7b87700-cb87-11ea-8abe-166fcade3a23.png)
![Screenshot 2020-07-21 at 19 22 27](https://user-images.githubusercontent.com/12850886/88080415-b5d6d400-cb87-11ea-93e6-5d7a0532261a.png)
![Screenshot 2020-07-21 at 19 21 58](https://user-images.githubusercontent.com/12850886/88080845-4b726380-cb88-11ea-8fba-9d6d81db059d.png)

ericharmeling commented on templates/_nav.html:
```diff
@@ -1,42 +1,47 @@
+  <div class="collapse navbar-collapse" id="navbarNavDropdown">
+    <ul class="navbar-nav">
+      <li class="nav-item">
+        <a class="nav-link" href="/">Home</a>
```
Note that https://github.com/cockroachlabs/movr-flask/pull/13 removes the Home page

ericharmeling commented on templates/rides.html:
```diff
@@ -1,38 +1,60 @@
+                    </form>
+                </div>
+            {% else %}
+                <div class="status unavailable">Unavailable</div>
```
> `<div class="status unavailable">Unavailable</div>`

This should be 

> `<div class="status unavailable">Inactive</div>`

since it's referring to the ride, and not the vehicle. (If the ride is inactive, the vehicle will be available).

ericharmeling commented on templates/vehicles.html:
```diff
@@ -1,43 +1,69 @@
+                    </div>
+
+                    <div class="row start-ride">
+                        <form class="form form-horizontal" method="POST" action="/rides/start/{{ vehicle.id }}">
```
> `<form class="form form-horizontal" method="POST" action="/rides/start/{{ vehicle.id }}">`

This should be 

> `<form class="form form-horizontal" method="POST" action="/vehicles/remove/{{ vehicle.id }}">`

since `form_v` is defined in the `user` route (in `server.py`) as `RemoveVehicleForm`.

elkmaster commented:
@ericharmeling, thank you for review, good catch. Fixed both issues, added changes from [#13](https://github.com/cockroachlabs/movr-flask/pull/13) including logic for active page class in nav.

ericharmeling commented:
@elkmaster 

Awesome. Thanks!

@celiala 

Before we merge this, could you look over the changes to see if they incorporate everything from #13? :)

celiala commented:
Haven't looked at the code at all -- but pulled the PR and played around with it.

This looks great, @elkmaster ! And confirming that this incorporates all the relevant logic from #13 (so I'll go ahead an abort that PR now).

celiala commented:
I'm not seeing Google Maps on my end (see snapshot). Wonder if this is just me, or if this is reproducible:

![image](https://user-images.githubusercontent.com/3051672/88214492-49f87800-cc28-11ea-975a-2b066ed6d7e6.png)

elkmaster commented:
> I'm not seeing Google Maps on my end (see snapshot). Wonder if this is just me, or if this is reproducible:
> 

Maybe you need enable embed \ geocoding  API's inside gcloud dashboard for your api key. Is there any errors in js console?

elkmaster commented:
Check if this API's are enabled for your gkey:

Geocoding API
Maps JavaScript API
Maps Embed API
